pkg/ingester/metrics.go

```diff
@@ -375,6 +378,15 @@ func newIngesterMetrics(
 			Name: "cortex_ingester_prepare_shutdown_requested",
 			Help: "If the ingester has been requested to prepare for shutdown via endpoint or marker file.",
 		}),
+
+		circuitBreakerTransitions: promauto.With(r).NewCounterVec(prometheus.CounterOpts{
+			Name: "cortex_ingester_circuit_breaker_transitions_total",
+			Help: "Number of times the circuit breaker has entered a state",
+		}, []string{"ingester", "state"}),
```

Comment:
All metrics in this file are ingester-specific already, why do we include `ingester` label here?

Reply:
Circuit breakers could be added to any other component. I think that it is important to understand if it is an ingester-specific circuit breaker and not for example querier-specific circuit breaker.

Reply:
I missed your comment before. While you're right that other components could use circuit breakers, they will likely use metric names without "ingester" in the name. If we need to have generic circuit breaker metrics in the future, we can consider adding `component` label to them.

Reply:
@pstibrany so you are proposing to remove the `_ingester` part from the metrics now and to potentially add the `component` label in the future?

Reply:
> @pstibrany so you are proposing to remove the `_ingester` part from the metrics now and to potentially add the `component` label in the future?

I'm fine keeping metric name as-is for now, we can make it more generic (and add `component`) when we add more CBs in the future.